nominees/operation-serenata-de-amor.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Operation Serenata de Amor",
+  "aliases": [
+    ""
+  ],
+  "description": "An open project that uses data science - the same technologies used by giants like Google, Facebook and Netflix - for the purpose of monitoring public spending and sharing information in a way accessible to everyone. Serenata has created Rosie: an artificial intelligence who analyzes Brazilian congresspeople expenses while they are in office. Rosie can find suspicious spendings and engage citizens in the discussion about these findings.",
+  "website": "https://serenata.ai/en/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/okfn-brasil/serenata-de-amor/blob/main/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 16,
+      "evidenceText": "Rosie can find suspicious spendings and engage citizens in the discussion about these findings. To allow people to visualize and make sense of data Rosie generates, we have created Jarbas. In this website users can browse congresspeople expenses and get details about each one of the suspicions.",
+      "evidenceURL": "https://serenata.ai/en/"
```

Comment:
It seems like there is a clear tie-in to the following [SDG 16 targets](https://sdgs.un.org/goals/goal16):

* **16.5** (_Substantially reduce corruption and bribery in all their forms_)
* **16.6** (_Develop effective, accountable and transparent institutions at all levels_)
* **16.10** (_Ensure public access to information and protect fundamental freedoms, in accordance with national legislation and international agreements_)

Instead of using the boilerplate text they use on their website for SDG relevance, could we ask for a justification in terms of one or more of these SDG 16 targets? The link seems clear, but it would be good to be explicit!